etna/articles/models.py

```diff
@@ -371,64 +368,13 @@ class PageGalleryImage(Orderable):
         help_text="An optional caption, which will be displayed directly below the image. This could be used for image sources or for other useful metadata.",
         blank=True,
     )
-    transcription_header = models.CharField(
-        verbose_name=_("transcription header"),
-        max_length=50,
-        default="Transcript",
-        help_text=_("Header for the transcription (max length: 50 chars)."),
-    )
-    transcription_text = RichTextField(
-        verbose_name=_("transcription text"),
-        features=["bold", "italic", "ol", "ul"],
-        max_length=1500,
-        help_text=_("An optional transcription of the image (max length: 1500 chars),"),
-        blank=True,
-    )
-    translation_header = models.CharField(
-        verbose_name=_("translation header"),
-        max_length=50,
-        default="Translation",
-        help_text=_("Header for the translation (max length: 50 chars)"),
-    )
-    translation_text = RichTextField(
-        verbose_name=_("translation text"),
-        features=["bold", "italic", "ol", "ul"],
-        max_length=1500,
-        help_text=_(
-            "An optional translation of the transcription (max length: 1500 chars)."
-        ),
```

Comment:
Note: If there's been population already it might be worth adding these to the migration fields, but that might more pain / time than it's worth.

Reply:
Yeah, it's a brand new page type that hasn't been used yet, so not worth the effort.